unicode-data/unicode-data.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.2
 name:                unicode-data
-version:             0.3.0
+version:             0.3.1
```

Comment:
The commit message seems to have a typo. Should be "Bump" instead of "Dump"?

Reply:
Good catch! I am going to squash the commit and merge.